openmc/deplete/nuclide.py

```diff
@@ -681,6 +681,11 @@ def __repr__(self):
         return "<{} containing {} products and yields>".format(
             self.__class__.__name__, len(self))
 
+    def __deepcopy__(self, memo):
+        result = FissionYield(self.products, np.copy(self.yields))
```

Comment:
To be consistent with other usage throughout the class
```suggestion
        result = FissionYield(self.products, self.yields.copy())
```

Reply:
Definitely! Changing this now.